### ISSUE TITLE
Enable Regular Beta Data Refresh

### ISSUE THREAD

**Issue by martinhannigan** [peeringdb/peeringdb#1534]
**Is your feature request related to a problem? Please describe.**

"This is a beta/testing instance of PeeringDB. Some features may work differently than production. Do not store data here that you intend to keep - automatically scheduled refreshes from production are currently disabled.

Database last synchronized at 2024-01-17 01:03:17"

Beta is poorly used. unfortunately. The net result is we send more bugs to production than we might if we had better use of beta. In order to do that Beta should be a mirror of production so that it _can_ be used as an operational interface with the caveat that users should question results. This is common with beta versions of products; users agree there is no liability of the organization if they use it and everyone YOLO's at their own risk and choice. New features usually drive the desire for beta access and some of what we're rolling out is attractive in that theme.

**Who is affected by the problem?**

Users.

**What is the impact?**

Participation of the community and committees in helping to ascertain quality.

**Are there security concerns?**

Unsure.

**Are there privacy concerns?**

No.

**Describe the solution you'd like**

Do a COGS analysis of refreshing and storing the data if the same data can't be pointed to
If there's no cost or minimal cost impact (what is that cost?) consider it
If it's inordinately expensive for some reason, discuss how to reduce cost or determine another solution

**Do you think this feature will require a formal design?**

Yes.

**Describe alternatives you've considered**

We could do nothing, but that doesn't advanced PDB.

**Could this feature request need support from the Admin Committee?**

They should be consulted. I'm not certain if it generates more, or less, ticketing. I would suggest it may be less since we could coral beta errors to a "report" function or an email address that goes directly to developer if it can be made to work effectively.

**What is the proposed priority?**

Medium

**Provide a rationale for any/all of the above**

Improving quality. Making everyone's life easier.

**Comment by mcmanuss8** [comment]
+1

**Comment by jackcarrozzo** [comment]
+1

**Comment by jbartig** [comment]
+1 👍

**Comment by grizz** [comment]
+1 to doing it hourly

**Comment by martinhannigan** [comment]
+1 make it more prod like. But Beta[tm]

**Comment by leovegoda** [comment]
**Summary**
1. Sync data from www.peeringdb.com to beta.peeringdb.com once an hour
2. Include a prominent note of when data was last updated
3. Include a prominent note that updates made via beta.peeringdb.com are not sync'd back to www.peeringdb.com

**Comment by martinhannigan** [comment]
Is it possible to neuter Beta "edit" instead with an error message?

Tks

**Comment by leovegoda** [comment]
> Is it possible to neuter Beta "edit" instead with an error message? Tks

Good idea. Add a closing note on edits made to test edit functionality on beta, so that users can both test update functionality but are told again that it needs to be repeated on www.peeringdb.com.

**Updated Summary**
1. Sync data from [www.peeringdb.com](http://www.peeringdb.com/) to beta.peeringdb.com once an hour
2. Include a prominent note of when data was last updated
3. Include a prominent note that updates made via beta.peeringdb.com are not sync'd back to [www.peeringdb.com](http://www.peeringdb.com/)
4. The change save dialog on beta.peeringdb.com should include a note that the change is only saved locally, reminding users that they are on beta and not the production service

**Comment by leovegoda** [comment]
The first step to completing this work is the development of a specification. That is being scheduled for 2.57.0, with implementation to be scheduled as soon after that as practical

**Comment by leovegoda** [comment]
Completed with 2.67.0
